### DISCUSSION TITLE
ScriptReference/UIElements.UQueryBuilder_1.Build

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#626] [Notes]
# ScriptReference/UIElements.UQueryBuilder_1.Build

https://docs.unity3d.com/ScriptReference/UIElements.UQueryBuilder_1.Build.html

**Comment by vertxxyz** [comment]
See [`UQueryState<T0>`](https://docs.unity3d.com/ScriptReference/UIElements.UQueryState_1.html).
